website/docs/cloud-docs/api-docs/policy-checks.mdx

```diff
@@ -251,16 +251,17 @@ The GET endpoints above can optionally return related resources, if requested wi
 | `run`              | The run this policy check belongs to.  |
 | `run.workspace`    | The associated workspace of the run.   |
 
+<!-- BEGIN: TFC:only name:opa-policies -->
 
-## List Policy Evaluations in the task stage
+## List Policy Evaluations in the Task Stage
```

Comment:
Ah, you commented this out - yay! :)  I think we want a line underneath the title that this is only for OPA policies. I assume that's why we commented it out, right?  Do we have an equivalent endpoint for Sentinel (sorry it's so hard to see from the GitHub preview) - if yes, maybe we be like "This endpoint is only available for OPA policies. For Sentinel policies, use the XX endpoint." 

Thoughts?

Reply:
I am not sure if there is an equivalent sentinel endpoint for policy outcomes. But it looks like `List Policy Checks` might be a single endpoint (from sentinel) that kind of does what both these endpoints do for OPA. I would probably avoid adding the For Sentinel policies, use the XX endpoint for now.